README.md

```diff
@@ -89,8 +89,6 @@ The tutorial!!
 Press ENTER to continue...
 ```
 
-If not, it's probably our fault. Please let us know by [filing an issue on the meeshkan-tutorial repo](https://github.com/meeshkan/meeshkan-tutorial/issues).
```

Comment:
I'd say keep this but changing the link to `https://github.com/meeshkan/meeshkan/issues`